src/Commands/Admin.py

```diff
@@ -200,6 +201,35 @@ async def forceReport(mentions: str, roles: List[Role], lbChannel: Channel, *arg
         )
 
 
+def multiplier(roles: List[Role], *arg) -> Embed:
+    if (Queue.isBotAdmin(roles)):
+        try:
+            value = float(arg[0])
+        except ValueError:
+            updateMMRMultiplier(1.0)
```

Comment:
Do we need to reset the multiplier to 1 if they mess up the command? Would it be better just leave whatever it is set to and just return the error?